website/docs/patterns/filter-patterns/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,134 @@
+
+### Data set
+
+A data set is a broad term for an array of items, objects, or related information presented as separate but related records. A data set is commonly represented via a [Table](/components/table) but may be expressed in other formats depending on the type of information.
+
+**Dataset represented by a table**
+
+![Tabular data set](/assets/patterns/filter-patterns/example-data-set-tabular-data.png)
+
+**Dataset represented by a grid**
+
+![Non-tabular data set](/assets/patterns/filter-patterns/example-data-set-non-tabular-data.png)
+
+### Filter positioning
+
+Filters are responsible for the functional aspect of limiting a data set and for displaying the [parameters](/patterns/filter-patterns?tab=core%20concepts#parameter) which a user can filter upon. They often consist of numerous parameters that mirror the columns of a table or metadata contained within the data set.
+
+Filter positioning is largely dependent on the complexity of the data set and the scope for which the filters are being applied (are the filters applied at the page level, or only to the data set they are paired with).
+
+#### Filter bar
+
+Organizing filters in a  **filter bar** is a common positioning method and consists of one or more dropdowns, buttons, or input components that are used to select values from a set of parameters. A [Segmented Group](/components/segmented-group) can be used within a filter bar to group similar parameters and support complex filtering.
+
+![Filter bar example](/assets/patterns/filter-patterns/filter-bar-segmented-group.png)
+
+**Ex 1.0: Horizontal filter bar**
+
+![Horizontal orientation of the filter bar](/assets/patterns/filter-patterns/layout-filter-bar.png)
+
+#### Sidebar
+
+If space permits, filters can be positioned vertically in a **sidebar**, either at the page level, or in-line with the data set.
+
+- **Page-level sidebar:** can be used for more complex data sets and if filtering contents of the entire page is required.
+- **In-line sidebars:** can be used when the number of filterable parameters is minimal and if space permits.
+
+**Ex 1.1: Page-level filter sidebar**
+
+![Vertical orientation of the filter bar](/assets/patterns/filter-patterns/layout-sidebar-left-page-level.png)
+
+**Ex 1.2: In-line filter sidebar**
+
+![Vertical orientation of the filter bar on the left](/assets/patterns/filter-patterns/layout-sidebar-left-inline.png)
+
+![Vertical orientation of the filter bar on the right](/assets/patterns/filter-patterns/layout-sidebar-right-inline.png)
+
+### Applied filters
+
+The **applied filters** displays which filter values have been applied (if any) and provides a user with a method to clear or reset all filters at once. In most circumstances the filters that have been applied should be represented using a [Tag](/components/tag) component which allows for the individual dismissal of a filter value.
```

Comment:
@jorytindall Was this meant to be resolved without applying the suggested changes? Just asking because there are a couple minor grammar and punctuation errors in the original.